packages/babel-plugin-proposal-private-methods/test/fixtures/accessors/tagged-template/input.js

```diff
@@ -0,0 +1,7 @@
+class Foo {
+  get #tag() {
+    return () => this;
+  }
```

Comment:
Seems we dropped the tagged template:
```suggestion
  }
  
  constructor() {
    this.#tag``;
  }
```